cmd/keytransparency-server/frontend.go

```diff
@@ -71,22 +71,22 @@ var (
 func openDB() *sql.DB {
 	db, err := sql.Open(engine.DriverName, *serverDBPath)
 	if err != nil {
-		log.Fatalf("sql.Open(): %v", err)
+		glog.Exitf("sql.Open(): %v", err)
```

Comment:
Why not use `glog.Fatalf` here? We're using it in `signer.go`. I'm fine with either if we don't care about the difference between the two, I'm just concerned about consistency.